### PR TITLE
Pin ffi use 1.14.1 as 1.14.2 contains LoadError bug on Windows

### PR DESCRIPTION
**Pull request by sophieyfang**
This PR is to fix:

```
C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': 126: The specified module could not be found.   - C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/ffi-1.14.2-x86-mingw32/lib/2.6/ffi_c.so (LoadError)
```

Related github issue: https://github.com/ffi/ffi/issues/868

I built an agent using this branch, tested the built agent in a new VM and verified the agent successfully send eventlog2 to the backend

image

### CODE REVIEW

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: windows-installer/generate_sdl_agent_exe.ps1 [hunk 1]
```diff
@@ -145,6 +145,8 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 $gem_installer = $SRC_ROOT + '\bin\gem_installer'
 $core_gems_rb = $SRC_ROOT + '\core_gems.rb'
 $plugin_gems_rb = $SRC_ROOT + '\plugin_gems.rb'
+# Pin ffi to version '1.14.1'. Remove the line when https://github.com/ffi/ffi/issues/868 is resolved.
+& $GEM_CMD install ffi --version '1.14.1'
```
[the remark this answers is not in the document]
Comment (sophieyfang) [reply windows-installer/generate_sdl_agent_exe.ps1]: good point. i was being blind

File: windows-installer/generate_sdl_agent_exe.ps1 [hunk 2]
```diff
@@ -145,6 +145,8 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 $gem_installer = $SRC_ROOT + '\bin\gem_installer'
 $core_gems_rb = $SRC_ROOT + '\core_gems.rb'
 $plugin_gems_rb = $SRC_ROOT + '\plugin_gems.rb'
+# Pin ffi to version '1.14.1'. Remove the line when https://github.com/ffi/ffi/issues/868 is resolved.
```
Comment (igorpeshansky) [inline windows-installer/generate_sdl_agent_exe.ps1:148]: By `the line`, did you mean `the following line`?
Optionally, I would avoid repeating the actual version in the comment, so I'd even make this something like:
`Pin ffi version until https://github.com/ffi/ffi/issues/868 is resolved.`

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
One minor tweak.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**